content/hi/partners/_index.html

```diff
@@ -0,0 +1,91 @@
+---
+title: सहयोगी
+bigheader: कूबरनेट्स सहयोगी
+abstract: कूबरनेट्स इकोसिस्टम का विस्तार.
+class: gridPage
+cid: सहयोगी
+---
+
+<section id="users">
+    <main>
+        <h5>कुबरनेट्स सहयोगियों संग एक मज़बूत, जीवंत कोड बेस का निर्माण करता है जो संपूरक मंचों के स्पेक्ट्रम को सहयोग करता है।</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>कुबरनेट्स प्रमाणित सेवा प्रदाता</b>
+              </h5>
+              <br>उद्यमों द्वारा कुबरनेट्स सफलतापूर्वक अपनाने में सहयोगी अनुभवी सेवा प्रदाता।
+              <br><br><br>
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">KCSP सहयोगी देखें</button>
+              <br><br><a href="https://www.cncf.io/certification/kcsp/">KCSP बनने में दिलचस्प</a>?
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>प्रमाणित कूबरनेट्स Distributions, Hosted Platforms, तथा Installers</b>
+              </h5>सॉफ्टवेयर अनुरूप चलने से ये सुनिश्चित किया जाता है कि प्रत्येक विक्रेता का संस्करण जरूरी API का समर्थन करे।
+              <br><br><br>
+              <button id="conformance" class="button" onClick="updateSrc(this.id)">अनुरूप सहयोगी देखें</button>
```

Comment:
This sentence is not matching up `अनुरूप सहयोगी देखें`.

Reply:
Please translate it for me!
"See Conformance partners"

Reply:
Translation of  `See Conformance partners` is `अनुरूपता भागीदार देखें`

How about `सहभागी`?

`अनुरूप सहभागी देखें`

Reply:
सहयोगी, भागीदार and सहभागी can all be used in place of partners, I thought the problem is with the whole sentence, So, I am concluding that there is no problem with the translation meaning.
Either way, I will take your suggestion and overwrite the original one.

Reply:
Thanks.